external/README.md

```diff
@@ -2,6 +2,25 @@
 
 Third Party container tests help verify that the adoptium binaries are *good* by running a variety of Java applications inside of Docker containers. adoptium/aqa-tests/[Issue #172](https://github.com/adoptium/aqa-tests/issues/172) lists the applications that we have initially targeted to best exercise the adoptium binaries.  For each application, we choose to run a selection of their functional tests.
 
+## Podman, docker and others. Sudo, runas and others
+
+The toolchain understands two environment variables:
+```
+EXTERNAL_AQA_RUNNER=docker|podman|...
+```
+defaults to podman if podman is installed, otherwise to docker
```

Comment:
Just to double check if default is docker or podman?

Reply:
If podman is installed, then podman is used. If there is only docker, docker is used. Logc for this is same as I did it in temurin-build.  Podman have docker bindings, so with podman command, you have also docker command, which calls podman. Those bindings are... misbihaving from time to time. Thus it is better to use directly podman when you know what to do.

No one sane will have both dockler and podman installed (if it is even possible and working).